### ISSUE TITLE
[Accessibility] Configurable aria properties/translations for CVCHint component

### ISSUE THREAD

**Issue by ryanolw** [Adyen/adyen-web#1346]
**Is your feature request related to a problem? Please describe.**

The current CVCHint component within the drop-in does not satisfy the success criterion defined in [WCAG Guideline(s): 1.1.1 non-text content](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html). The CVC image should either be labelled by a text alternative, or not focused on by screen readers (via aria-hidden).

**Describe the solution you'd like**

Expose a translation field `creditCard.encryptedSecurityCodeImage.aria.label` that will be used to populate `aria-labelledby` for the CVCHint container (likely https://github.com/Adyen/adyen-web/blob/master/packages/lib/src/components/Card/components/CardInput/components/CVCHint.tsx#L14).

**Describe alternatives you've considered**

**Additional context**
Screenreader prompts "missing image descriptions" upon focusing on CVC image
https://user-images.githubusercontent.com/93317277/139386402-5f2627af-8709-4a8c-bbf3-3a168b417f41.mov

Happy to provide more information if required.

**Comment by ribeiroguilherme** [comment]
Hi @ryanolw ,

Thanks for reaching out to us. We are working on a11y issues, and this problem is going to be addressed on #1353 .
I will update this ticket as soon as we released the fix.

**Comment by ribeiroguilherme** [comment]
@ryanolw we released the version 5.2.1 which contains the fix for this issue. Thanks again for reaching out!
